packages/mobile/src/harmony-native/components/layout/Flex/Flex.tsx

```diff
@@ -12,7 +12,7 @@ export const Flex = styled(Box, {
 })<NativeFlexProps>((props) => {
   const {
     theme,
-    direction,
+    direction = 'row',
```

Comment:
I think the default for flex components on mobile is column. Not sure if this will be counterintuitive for people using this component on mobile, but I agree with this for parity with web

Reply:
good callout, hmm i kinda like things to be similar, maybe we can change webs to be columb by default and then we update this or something?

Reply:
going to undo this now though so it doesn't mess up the other flex items